app/javascript/views/userDetail/NoPermissionsWarning.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { Modal, ModalBody, ModalFooter, ModalHeader, Button } from '@cwds/components'
+
+export default class ModalComponent extends Component {
```

Comment:
I think this class should be named after the file it's in.  It's not important that it's a ModalComponent (and we might be defining the same name other places, if it were). What we'll want to know down the road is that it's this NoPermissionsWarning object
```suggestion
export default class NoPermissionsWarning extends Component {
```